docs/manual/forms/variables.rst

```diff
@@ -0,0 +1,170 @@
+                                                         ``pseudo`` heeft. Anders leeg.
+auth_context       object    ``{"source": "...", ...}``  De volledige authenticatiecontext, met machtigingsinformatie. Zie
+                                                         :ref:`manual_forms_variables_auth_context` voor de beschrijving.
+================== ========= =========================== =========================================================================
+
+**Verouderde variabelen**
+
+Deze variabelen zijn nog wel beschikbaar, maar we raden aan om deze niet meer te
+gebruiken. In versie 3.0 van Open Formulieren kunnen deze verwijderd worden.
+
+=============== ========= =========================== =========================================================================
+Variabele       Type      Voorbeeldwaarde             Toelichting
+=============== ========= =========================== =========================================================================
+auth.machtigen  object                                Verouderd, wordt in 3.0 verwijderd.
+auth.attribute  string    ``bsn``                     Kan de waarden ``bsn``, ``kvk`` of ``pseudo`` hebben (verouderd,
+                                                      gebruik bij voorkeur ``auth_type``).
+=============== ========= =========================== =========================================================================
+
+.. _manual_forms_variables_auth_context:
+
+Authenticatiecontext
+--------------------
+
+De "authenticatie context" bevat alle beschikbare informatie over *hoe* iemand ingelogd
+is op een formulier. Deze bundel informatie is beschikbaar in de vaste variabele
+``auth_context``.
+
+Wanneer er niet ingelogd is op het formulier, dan is de waarde van deze variabele
+``null``.
+
+.. note:: Het is een nieuwere en volledigere variant van de ``auth`` variabele.
+   Er zijn geen plannen om deze laatste te verwijderen.
+
+De variabele bevat een bak aan informatie, gestructureerd volgens het
+authenticatiecontextdatamodel (TODO: add link). De structuur is als volgt:
+
+.. code-block:: json
+
+    {
+        "source": "string",
+        "levelOfAssurance": "string",
+        "representee": {
+            "identifierType": "string",
+            "identifier": "string"
+        },
+        "authorizee": {
+            "legalSubject": {
+                "identifierType": "string",
+                "identifier": "string"
+            },
+            "actingSubject": {
+                "identifierType": "string",
+                "identifier": "string"
+            }
+        },
+        "mandate": {
+            "role": "string",
+            "services": [
+                {
+                    "id": "string",
+                    "uuid": "string"
+                }
+            ]
+        }
+    }
+
+Merk op dat niet alle attributen aanwezig zijn, dit hangt af van het inlogmiddel (
+DigiD, eHerkenning) en of er wel/niet sprake is van een machtiging én de soort
```

Comment:
```suggestion
Merk op dat niet alle attributen aanwezig zijn, dit hangt af van het inlogmiddel (DigiD, eHerkenning) en of er wel/niet sprake is van een machtiging én de soort
```

Reply:
I don't see the difference or mistake :sweat_smile: